api/src/main/resources/messages.properties

```diff
@@ -663,12 +663,6 @@ providermanagement.assignPatientToProvider=Assign Patient to Provider
 
 pih.app.cohortBuilder.label=Cohort Builder
 
-pih.app.bloodPressure.obsTable.title=Blood Pressure Observations
```

Comment:
There's a couple new codes and a couple removed, but mostly this a reorder and alphebetization.